pom.xml

```diff
@@ -63,6 +63,7 @@
         <module>afs-security-analysis-local</module>
         <module>afs-storage-api</module>
         <module>afs-ws</module>
+        <module>afs-spring-server</module>
```

Comment:
could use the new powsybl-parent-ws in this pom ? It allows to build a docker image and the spring exec jar

Reply:
Do you propose to use it for the whole powsybl-afs project ?

I wonder if it's a good idea, since not all modules of the project are "webservice-related". I understand there would be a (small) added value though. Maybe to be decided in a separate issue ?

Reply:
yes, on the root pom. It is designed to be used like that: it doesn't do anything by itself, plugins have to be declared in the one pom that does use spring and docker images.

The reason we designed it like this is to "workaround" the following maven limitation. When you are doing a multimodule build with maven, most builds make all the poms inherit from the pom that is in their parent directory (because that's the only way in maven to share things like properties and many others). So it's really painful if one of your poms in the hierarchy doesn't inherit from the parent dir, and instead inherits from an external pom like powsybl-parent-ws. People will not expect that and be surprised when this subproject doesn't share the configuration of the rest (and the only solution is to duplicate everything).

Reply:
After more thought, I think it will not be really useful in that PR, again because the module does not provide a runnable application, but only spring components. Maybe it could be useful for 3rd party projects using those components, but in that case I think it's the coice of that project to use the powsybl-parent-ws.